src/main.py

```
@@ -316,11 +456,19 @@ def get_current_url(self, display_id):
     
     def cleanup(self):
         """Clean up Chrome processes"""
+        # Attempt to resume displays first, in case they were suspended
+        try:
+            logging.info("Attempting to resume displays as part of cleanup...")
+            self.resume_displays() # resume_displays checks xrandr_state internally
+        except Exception as e:
+            logging.error(f"Error resuming displays during cleanup: {e}")
+            # Continue with cleanup even if display resumption fails
+
         if hasattr(self, '_cleaning_up') and self._cleaning_up:
```

Pylint (3.3.7)

[error] Access to member '_cleaning_up' before its definition line 470

(E0203)